outputs.tf

```diff
@@ -46,3 +46,8 @@ output "kms_public_endpoint" {
   description = "Key Management Service instance public endpoint URL"
   value       = local.kms_public_endpoint
 }
+
+output "cbr_rule_ids" {
+  description = "CBR rule ids created to restrict KMS(Key Protect)"
+  value       = module.cbr_rule[*].rule_id
```

Comment:
Description should say `CBR rule ids created to restrict Key Protect` as the CBR rules currently do not apply to HPCS. Also change the output value to be `one(module.cbr_rule[*].rule_id)` so it will be null whenever there is nothing to output rather than an empty array